components/core/src/os/net/mod.rs

```diff
@@ -22,3 +22,76 @@ mod imp;
 mod imp;
 
 pub use self::imp::*;
+
+extern crate dns_lookup;
```

Comment:
With recent changes to Rust for the 2018 Edition, `extern crate` is [hardly ever required](https://doc.rust-lang.org/edition-guide/rust-2018/module-system/path-clarity.html#no-more-extern-crate) anymore.